src/components/icon/_appstore.less

```diff
@@ -1,9 +1,11 @@
 .appstore-btn {
   box-shadow: 0px 1px 4px #4d4d4d;
+  background-color: #000000;
+  color: #fff;
   transition: all 0.2s;
   position: relative;
   padding-top: 15px;
-  padding-left: 60px;
+  padding-left: 30px;
```

Comment:
The `padding-left: 30px` only affects `.block-btn` because of line 31 hence for other buttons it is `50px`